app/controllers/application_controller.rb

```diff
@@ -1,4 +1,9 @@
-# frozen_string_literal: true
-
 class ApplicationController < ActionController::API
+  before_action :authenticate_request
+  attr_reader :current_user
+
+  def authenticate_request
+    @current_user = AuthorizeApiRequest.call(request.headers).result
```

Comment:
Where is located this `AuthorizeApiRequest` class?

Reply:
That's the part of the token I'm working on. I'm having some trouble generating it. I thought I left that part out.